L/libCEED/build_tarballs.jl

```diff
@@ -30,4 +30,4 @@ products = [
 dependencies = Dependency[]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, preferred_llvm_version=v"11")
```

Comment:
Nevermind, with #3288 it shouldn't be necessary anymore:
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
```
Let's try